internal/v1/handler_blueprints.go

```diff
@@ -39,6 +39,12 @@ func (u *User) CryptPassword() error {
 	if u.Password == nil || (len(*u.Password) == 0 || crypt.PasswordIsCrypted(*u.Password)) {
 		return nil
 	}
+
+	if u.IsRedacted() {
+		return errors.New(
+			"password is redacted and thus can't be hashed and stored in database, use plaintext password or already hashed password",
+		)
+	}
```

Comment:
Sort of fails the use case when a user would really really want `<REDACTED>` as a password. A weird one, granted. Just saying, I like what I am seeing here. Nicely done.